### ISSUE TITLE
Way to See Extended Keys, Paths, and Address Encoding for Existing Wallet via GUI?

### ISSUE THREAD

**Issue by mflaxman** [cryptoadvance/specter-desktop#332]
This page on wallet creation looks great:
![Screen Shot 2020-08-30 at 12 29 56 PM](https://user-images.githubusercontent.com/448644/91670165-987b1b00-eae0-11ea-8177-74dc3f5a1de3.png)

When you want to verify the info on (at least a quorum of) your devices it's harder to find. This is decent but it shows devices (not keys) and doesn't give you your `m-of-n`:
![coordinate-multisig-view-policy-specter](https://user-images.githubusercontent.com/448644/91670685-48528780-eae5-11ea-98cb-36b6eeb63eb4.png)

You can clickthrough and see the key details for that device, but it might have many keys (due to derivation paths, address encoding, etc). The `Export Wallet` functionality is good, but a script descriptor is too complicated for 99% of GUI users.

Basically, I'm just asking for a GUI view similar to the one on wallet creation.

Is that something you'd consider adding?

Thanks!

**Comment by ben-kaufman** [comment]
Hi, yes that definitely makes sense. I've opened a PR #335 to add such list to the settings page, let me know what you think and if that's about what you had in mind.
